integration-tests/spring-data-jpa/src/test/java/io/quarkus/it/spring/data/jpa/PostResourceTest.java

```diff
@@ -79,9 +79,20 @@ void testAllPostComment() {
                 .body("size()", is(2));
     }
 
-    //    Pay attention to the order, this test must be executed always the last one
     @Test
     @Order(7)
+    void testFindPostByTitleContainingText() {
+        Post post = when().get("/post/mandatory/1").then()
+                .statusCode(200)
+                .extract().body().as(Post.class);
+
+        assertThat(post).isNotNull();
+        assertThat(post.getId()).isEqualTo(1L);
+    }
+
+    //    Pay attention to the order, this test must be executed always the last one
```

Comment:
I know this isn't because of this PR but since you moved it: I think this comment no longer makes sense as this is definitely not the last test being executed

Reply:
good catch, thanks!